plugin.video.nhlgcl/resources/lib/nhl_tv.py

```diff
@@ -473,115 +485,115 @@ def fetchStream(game_id, content_id,event_id):
     i=0
     for i in range (0,9):
         #Org
-        url = 'https://mf.svc.nhl.com/ws/media/mf/v2.4/stream?contentId='+content_id+'&playbackScenario=HTTP_CLOUD_TABLET_60&platform='+PLATFORM+'&sessionKey='+urllib.quote_plus(session_key)        
-        req = urllib2.Request(url)       
+        url = 'https://mf.svc.nhl.com/ws/media/mf/v2.4/stream?contentId='+content_id+'&playbackScenario='+PLAYBACK_SCENARIO+'&platform='+PLATFORM+'&sessionKey='+urllib.quote_plus(session_key)
+        req = urllib2.Request(url)
         req.add_header("Accept", "*/*")
         req.add_header("Accept-Encoding", "deflate")
```

Comment:
I don't see that you decompress deflate-encoded response anywhere in your addon.